plugins/GSdx/GSTexture11.h

```diff
@@ -34,6 +34,11 @@ class GSTexture11 : public GSTexture
 	CComPtr<ID3D11RenderTargetView> m_rtv;
 	CComPtr<ID3D11DepthStencilView> m_dsv;
 
+	bool m_generate_mipmap;
```

Comment:
This isn't initialised in the constructor. Actually, I'd suggest removing `m_generate_mipmap` since it currently isn't used.

Apart from that the code looks ok.